Gems/Atom/RPI/Code/Source/RPI.Public/Pass/RenderPass.cpp

```diff
@@ -44,6 +44,10 @@ namespace AZ
             {
                 m_flags.m_bindViewSrg = true;
             }
+            if (passData && passData->m_deviceIndex > 0 && passData->m_deviceIndex < RHI::RHISystemInterface::Get()->GetDeviceCount())
```

Comment:
Can you elaborate on the condition here? Why do you need to check larger than `GetDeviceCount()`?